packages/gatsby-plugin-mdx/utils/mdx.js

```diff
@@ -1,5 +1,5 @@
 const mdx = require(`@mdx-js/mdx`)
-const grayMatter = require(`gray-matter`)
+const grayMatter = require(`./gray-matter`)
```

Comment:
I would name it different, otherwise other devs are expecting the original GrayMatter functionality here. 

one possible naming
```suggestion
const grayMatterWrapper = require(`./gray-matter-wrapper`)
```

Reply:
Thank you for the suggestion. I've updated.